pygmo/core.cpp

```diff
@@ -885,8 +892,80 @@ BOOST_PYTHON_MODULE(core)
 
     moead_.def("get_seed", &moead::get_seed);
 
+    // Exposition of various structured utilities
+    // Hypervolume class
+    bp::class_<hypervolume>("hypervolume", "Hypervolume class.", bp::no_init)
+        .def(bp::init<population, const bool>((bp::arg("pop"), bp::arg("verify") = true)))
```

Comment:
Any reason why this constructor is not in the line above, in place of the ``bp::no_init``? Also the ``const bool`` looks a bit unnecessary, ``bool`` alone is enough (in the original C++ ctor as well).